src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -774,5 +774,10 @@ function shouldHide(event: KeyboardEvent, keybindingService: IKeybindingService)
 	if (keybindingService.inChordMode || isValidChord) {
 		return false;
 	}
-	return event.key.length === 1 && !event.ctrlKey && !event.altKey && !event.metaKey && !event.shiftKey;
+	return isLetter(event.keyCode) && !event.ctrlKey && !event.altKey && !event.metaKey && !event.shiftKey;
+}
+
+function isLetter(code: number): boolean {
+	const key = String.fromCharCode(code);
```

Comment:
is there a better way to do this? I realize `keyCode` is deprecated

Reply:
This might work?

```
event.code.match(/^Key{A-Z}$/)
```

I think this contains a complete list of code values, so you could match what you want from there (eg. `Digit1`, `Equal`, ...):

https://developer.mozilla.org/en-US/docs/web/api/ui_events/keyboard_event_code_values